test/integration/features/basic.feature

```diff
@@ -27,6 +31,8 @@ Feature: Basic
       hello
       """
 
+  Scenario: Minishift --docker-env works properly
```

Comment:
Yes, we had done similar thing in ADB for service-manager tests cases - https://github.com/projectatomic/vagrant-service-manager/tree/master/features